### PR TITLE
SmileWanted Bid Adapter : support of eids

### PR DESCRIPTION
**Pull request by MaxSmileWanted**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Adding of support eids and tests

### CODE REVIEW

File: modules/smilewantedBidAdapter.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { isArray, logError, logWarn, isFn, isPlainObject } from '../src/utils.js';
 import { Renderer } from '../src/Renderer.js';
 import { config } from '../src/config.js';
+import { createEidsArray } from './userId/eids.js';
```
Comment (patmmccann) [inline modules/smilewantedBidAdapter.js:4]: please don't import from other modules, the eids are available on the bid request object

**Review by jlquaccia — CHANGES_REQUESTED** [review]
Once the logic for getting the eids is updated, think this looks good.

**Comment by MaxSmileWanted** [comment]
Hello,

We have updated the commit with modifications asked.

**Review by jlquaccia — APPROVED** [review]
LGTM

**Comment by muuki88** [comment]
Docs: https://github.com/prebid/prebid.github.io/pull/4289

**Comment by MaxSmileWanted** [comment]
Can you please merge it to have this modification in the 7.36.0

**Pull request merged**